src/test/java/com/google/sps/UserTest.java

```diff
@@ -39,6 +39,6 @@ public void userNotEquals() {
     User userA = new User("John");
     User userB = new User("Not John");
 
-    Assert.assertFalse(userA.equals(userB));
+    Assert.assertFalse((userA).equals(userB));
   }
 }
```

Comment:
Can you add tests for all the new methods?

Reply:
Done